docs_src/design/ucr/Global Common Configuration.md

```diff
@@ -0,0 +1,169 @@
+## Global Common Configuration 
+### Submitters
+- Lenny Goodell (Intel)
+
+## Change Log
+- [pending](URL of PR) (2022-11-??)
+
+### Market Segments
+- All
+
+### Motivation
+Currently the configuration for each of the EdgeX services have many common settings.  Most of these common settings have the same value for every service deployed in a single EdgeX based solution and possible across identical deployments of the same solution. The motivation for the UCR is to reduce this redundancy allowing for better manageability of service configuration across EdgeX services.

```

Comment:
If I read this UCR correctly, this motivation doesn't fully capture the intent.  My interpretation of the UCR is that we are striving for DRY (don't repeat yourself) in the source code/configuration for EdgeX.  
Each service is allowed to have its own copy of the global configuration, but to affect a configuration change, we only want to have to change it in at most one place.  Right?

Reply:
Yes, eliminate duplication of setting as much as possible and ideally only change the common setting in one location,